### DISCUSSION TITLE
Convert html template to Vanjaro theme

### DISCUSSION THREAD

**Discussion by Samanka** [vanjarosoftware/Vanjaro.Platform#1206] [General]
Hi
I wanted to see,how to Convert Html templates to vanjaro platform theme
On a vanjaro project that I work on has a basic template
I want to upload more templates
How to Convert html templates to vanjaro format
Thank you for guiding me

**Comment by meetmandeep** [comment]
> how to Convert Html templates to vanjaro platform theme

You can easily build any design visually without any code in Vanjaro. After Vanjaro officially releases, we'll have webinars and tutorials on template designs. Vanjaro will have its own app store where vendors will be able to sell their templates.

**Reply by Samanka** [reply]
I realized that it is possible to create a website without code,
Notice that there is a default template in the Portals\0\vThemes\Basic folder right now.
I want to create more templates, I actually want to convert html files to templates and save
I want to know how you made this basic template
Please give a tutorial on this

**Reply by meetmandeep** [reply]
Default template is DNN core functionality and not really used in Vanjaro. We'll have tutorial and webinar on how to create templates for Vanjaro in future. 

For now, you can design a site and export it from Tools > Sites. However, there is no UI available to import a site yet; we're working on that.
